core/chains/evm/gas/block_history_estimator.go

```diff
@@ -976,3 +974,29 @@ func (b *BlockHistoryEstimator) EffectiveTipCap(block evmtypes.Block, tx evmtype
 		return nil
 	}
 }
+
+// Int64ToHex converts an int64 into go-ethereum's hex representation
+func Int64ToHex(n int64) string {
+	return hexutil.EncodeBig(big.NewInt(n))
```

Comment:
This seems like a roundabout way to format an int64 as hex. How about something more direct?
```suggestion
// Int64ToHex formats an int64 as a hex string with 0x prefix.
func Int64ToHex(n int64) string {
	return fmt.Sprintf("0x%x", n)
```